tests/integration/test_kfp_functional_v1.py

```diff
@@ -26,9 +28,39 @@
 from pytest_operator.plugin import OpsTest
 
 
+KFP_SDK_VERSION = "v1"
 log = logging.getLogger(__name__)
 
 
+# ---- KFP SDK V1 fixtures
+@pytest.fixture(scope="function")
+def upload_and_clean_pipeline_v1(kfp_client: kfp.Client):
+    """Upload an arbitrary pipeline and remove after test case execution."""
+    pipeline_upload_response = kfp_client.pipeline_uploads.upload_pipeline(
+        uploadfile=SAMPLE_PIPELINE["v1"], name=SAMPLE_PIPELINE_NAME
```

Comment:
You could use `KFP_SDK_VERSION` here, right?
```suggestion
        uploadfile=SAMPLE_PIPELINE[KFP_SDK_VERSION], name=SAMPLE_PIPELINE_NAME
```